app/src/main/java/com/bachelor_group54/funnregistrering/FragmentEnkeltFunn.java

```diff
@@ -280,6 +312,91 @@ public void updateFind() {
         funn.setFylke(fylke.getText().toString());
     }
 
+    public void pdfGenerator(){
+        // Creation of an object variable for the PDF document
+        PdfDocument pdfDocument = new PdfDocument();
+
+        //Paint is used to draw shapes and add text
+        Paint logo = new Paint();
```

Comment:
Gjør  du noe med logo eller er det bare input i linje 339 (canvas.drawBitmap)

Reply:
Er for øyeblikk kun input for å sette bakgrunn.